README.rst

```diff
@@ -141,6 +141,15 @@ Please note the following guidelines for contributing:
   * If you are adding new functionality, you must include basic tests
     and documentation.
 
+
```

Comment:
Thanks a lot! 

There's one more thing I think is needed, I can take it over if you like, or you can finish it off, up to you. :-)

We should at the least be using Django's own `TestCase` rather than `unittest.TestCase`, the former presumably has some Django specific setup. In fact if you look at Mezzanine, all the tests throughout it use Mezzanine's own `TestCase` (which extends Django's), and it provides the user creation/removal in setup/teardown, which would mean you can delete some of that duplicate code here. You can see an example used here: https://github.com/stephenmcd/mezzanine/blob/master/mezzanine/core/tests.py